### PR TITLE
Add SimplePopup and Overlay Components

### PR DESCRIPTION
**Pull request by sscholle**
## Proposed changes

- Adds Overlay (a dependency on SimplePopup)
- Adds SimplePopup

## Types of changes

What type of changes does your code introduce?
_Put an `x` in the boxes that apply, leave blank if none_

- [ ] Frontend
- [ ] Backend
- [ ] Bugfix
- [ ] New feature
- [ ] Component of Major Release
- [ ] Project Management
- [ ] Quality of Life

## Checklist

_Put an `x` in the boxes that apply._

- [ ] Local Cypress Tests Pass with my changes (`yarn cypress:run`)
- [ ] Lint and unit tests pass locally with my changes (`yarn lint && ./vendor/bin/phpunit`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Dependant PRs
 - #21 

## Relevant Issues
- https://github.com/bonlineza/objectives-and-key-results/issues/18
- https://github.com/bonlineza/objectives-and-key-results/issues/17
- https://github.com/bonlineza/objectives-and-key-results/issues/2

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,7 +7,7 @@
 /coverage
 
 # production
-/build
```
Comment (robguy21) [inline .gitignore]: i don't think we should be committing our builds, i think that when we deploy a new version we should build and deploy there.

If we include built files we're going to get useless merge conflicts on nearly every pr

File: index.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import AsideSlide from './src/components/AsideSlide/index.js';
+import DropOptions from './src/components/DropOptions/index.js';
+import Overlay from './src/components/Overlay/index.js';
+import SimplePopup from './src/components/SimplePopup/index.js';
+
+exports.AsideSlide = AsideSlide;
+exports.DropOptions = DropOptions;
+exports.Overlay = Overlay;
+exports.SimplePopup = SimplePopup;
```
Comment (robguy21) [inline index.js]: this is confusing- we're mixing

`require()` and `import` syntax here-

```
# es6
import a from b

export a;
export default a;

# commonjs
const a = require('b');

module.exports.a = a;
module.exports = a;
```

File: .gitignore (hunk 1 above)
Comment (sscholle) [reply .gitignore]: can you add a PR for instructions for deploying?

Comment (robguy21) [reply .gitignore]: We can either use GitHub releases or go back to the npm releases, but committing built files is a no-go.

I'll work on the doc for publishing tomorrow.

**Comment by pruhesi** [comment]
This change to the simple popup only exists in assa.
https://github.com/bonlineza/assa/commit/5dd36034491e24a73fc28d5abf63b4938bfc0d60

File: src/components/Overlay/scss/Styles.scss [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    @extend %modal;
+
+    & > .modal__inner {
+      min-width: 50vw;
+      min-height: 50vh;
+    }
+  }
+
+  .modal__inner {
+    // @include soft--small--sides;
```
Comment (pruhesi) [inline src/components/Overlay/scss/Styles.scss]: Are we gonna keep these commented out?

File: README.md [hunk 4]
```diff
@@ -54,3 +54,10 @@ yarn run build:publish
 ``` 
 
 *Note: you will be prompted to login with your username/email/password and for the new version number at the CLI
+
+## BUILD RELEASE
+```bash
+yarn global add rollup
+rollup -c
```
Comment (robguy21) [inline README.md]: this can be removed as per the other pr right?

**Pull request merged**